### ISSUE TITLE
Error elements within a wrapper ad

### ISSUE THREAD

**Issue by wroberts15** [tbuchok/vast-xml#27]
When creating a vast xml response for wrapper ads, setting error does not generate an error element

    ad = vast.attachAd({
      id : "theID"
     , structure : 'wrapper'
     , AdSystem : { name : 'AdSys', version : '2.0' }
     , AdTitle : 'WrapperAd'
     , Error : "http://the.error.com/err" // this line does not generate error element
     , VASTAdTagURI : vastResponseURL
    });

**Comment by tbuchok** [comment]
and for what it's worth, it looks like this isn't expected in a wrapper. i don't think this was intended in the spec. thoughts?

you can checkout the `wrapper-error` branch i just committed and then run:

```
$ xmllint -noout --schema test/files/vast3_draft.xsd test/files/wrapper.xml 
test/files/wrapper.xml:8: element Error: Schemas validity error : Element 'Error': This element is not expected. Expected is one of ( Impression, Creatives ).
test/files/wrapper.xml fails to validate
```

**Comment by wroberts15** [comment]
hm I was just looking at 2.0 spec and I'm pretty sure it was allowed, so maybe that's deprecated in 3?

**Comment by tbuchok** [comment]
no i think you're right http://www.iab.net/media/file/VASTv3.0.pdf (page 20)

sometimes the .xsd validation is wrong, but it's hard to tell. do you have a use case in production that you're working towards?

that'd help to figure out if its the right fit for the lib to do something outside of the schema validation.

**Comment by tbuchok** [comment]
my bad here, didn't read the error message -- this is all good. will merge and update npm package.

**Comment by tbuchok** [comment]
published to npm => vast-xml@1.1.0

**Comment by wroberts15** [comment]
Awesome thanks
